glm/gtx/quaternion.inl

```diff
@@ -230,7 +230,7 @@ namespace glm
 	{
 		mat<3, 3, T, Q> Result;
 
-		Result[2] = -normalize(direction);
+		Result[2] = -direction;
 		Result[0] = normalize(cross(up, Result[2]));
```

Comment:
do we really need this normalize as well since the line below also uses cross product without normalize and assuming that the provided up vector is already normalized?